apps/laboratory/src/utils/DataUtil.ts

```diff
@@ -61,6 +66,11 @@ export const ethersSdkOptions = [
     title: 'Without Email',
     link: '/library/ethers-no-email/',
     description: 'Configuration using ethers without email'
+  },
+  {
+    title: 'Without Socials',
+    link: '/library/ethers-no-socials/',
+    description: 'Configuration using ethers without socials'
```

Comment:
@magiziz Can you add for ethers5 and solana as well as we have email feature for those?

Reply:
Added 🙏